carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/application/RecruitmentService.kt

```diff
@@ -19,7 +21,7 @@ import kotlinx.serialization.*
 @DependentServices( StudyService::class )
 interface RecruitmentService : ApplicationService<RecruitmentService, RecruitmentService.Event>
 {
-    companion object { val API_VERSION = ApiVersion( 1, 0 ) }
+    companion object { val API_VERSION = ApiVersion( 1, 1 ) }
```

Comment:
The next CARP core version will be 1.2. I think the intent was to keep the versioning aligned with the version it was upgraded.